src/pages/HomePage.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import './HomePage.css';
+
+export default function HomePage() {
+  return (
+    <div className="whole-page">
```

Comment:
I would use semantic HTML here. This will improve readability and accessibility. Something like:

```jsx
<section className="home-page">
  <h1>Here's our homepage! Go to your list or add an item below.</h1>
  <nav>
    <ul>
      <li>
        <NavLink className="left-tab" to="/add-item">
          to add item
        </NavLink>
      </li>
      <li>
        <NavLink className="right-tab" to="/list">
          to list
        </NavLink>
      </li>
    </ul>
  </nav>
</section>
```

You could even create a class like `nav-item` that could applied to both links to hold their shared styles.